parachain/runtime/testnet-kintsugi/src/lib.rs

```diff
@@ -1724,6 +1724,18 @@ impl_runtime_apis! {
                 amount_1_min
             )
         }
+
+        fn calculate_remove_liquidity(
```

Comment:
Is this part of the runtime? I.e. do we need a runtime upgrade for this to work?

Reply:
I am not so sure, this _is_ the runtime api. It seems to me like it should be part of the runtime since you can't just call any arbitrary runtime code from the node - it needs explicit entry point which is the runtime api's job to provide